src/Functions/FunctionsStringBytes.cpp

```diff
@@ -0,0 +1,142 @@
+#include <cmath>
+#include <Columns/ColumnConst.h>
+#include <Columns/ColumnString.h>
+#include <Columns/ColumnVector.h>
+#include <DataTypes/DataTypeString.h>
+#include <DataTypes/DataTypesNumber.h>
+#include <Functions/FunctionFactory.h>
+#include <Functions/FunctionsStringBytes.h>
+#include <Functions/IFunction.h>
+#include <Common/BitHelpers.h>
+#include <Common/PODArray.h>
+
+namespace DB
+{
+
+inline UInt8 countBits(UInt64 x)
+{
+#if defined(__clang__)
```

Comment:
We only use clang, so there is no problem if we keep only this branch.

Reply:
There is `std::popcount`

Reply:
I now put std::popcount in the else branch.